src/context_store/dashboard/websocket_manager.py

```
@@ -0,0 +1,82 @@
+            return_exceptions=True,
+        )
+
+    async def start_consumer(self) -> None:
+        """Start consuming messages from the queue and broadcasting."""
+        while True:
+            try:
+                msg = await self._queue.get()
+                await self.broadcast("logs", msg)
+            except asyncio.CancelledError:
+                break
+            except Exception as exc:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)